raiden/api/validations/channel_validator.py

```diff
@@ -118,3 +118,31 @@ def can_close_channel(token_address,
         )
 
         return channels_to_close
+
+    @staticmethod
+    def can_settle_channel(token_address,
```

Comment:
I don't like the idea of a function called can_X that doesn't return a boolean. Maybe get_channels_to_settle or something like would make more sense in my opinion.